patches/0010-LPS-136119-Set-id-on-first-render-instead-of-changin.patch

```diff
@@ -0,0 +1,37 @@
+From a2ebadce31c58f4a725f95905ef97b6a4146d37f Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Marko=20=C4=8Ciko=C5=A1?= <marko.cikos@liferay.com>
+Date: Mon, 9 Aug 2021 18:04:44 +0200
+Subject: [PATCH] LPS-136119 Set `id` on first render, instead of changing it
+ with each render
+
+---
+ plugins/button/plugin.js | 9 ++++++++-
+ 1 file changed, 8 insertions(+), 1 deletion(-)
+
+diff --git a/plugins/button/plugin.js b/plugins/button/plugin.js
+index 9cc0584169..e0c996e9e4 100644
+--- a/plugins/button/plugin.js
++++ b/plugins/button/plugin.js
+@@ -143,7 +143,6 @@
+ 			}
+ 
+ 			var env = CKEDITOR.env,
+-				id = this._.id = CKEDITOR.tools.getNextId(),
+ 				stateName = '',
+ 				command = this.command,
+ 				// Get the command name.
+@@ -151,6 +150,14 @@
+ 				keystroke,
+ 				shortcut;
+ 
++			var id = this._.id;
```

Comment:
Solution seems to be pretty simple, so LGTM.

Reply:
I think you can try sending this solution to solve [this](https://github.com/ckeditor/ckeditor4/issues/4639) opened ticket in CKEditor's repo too